server/middlewares/database.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+
+const DB_HOST ='localhost';
+const DB_PORT ='27017';
+const DB_NAME ='dev-kanban';
+const DB_USER_NAME='';
+const DB_PASSWORD ='';
```

Comment:
We may need to move these details to the .env file but it can be handled later as the project proceeds